compiler/parser/chapel.lex

```diff
@@ -884,6 +948,55 @@ static void addCharEscape(char c) {
   }
 }
 
+// Convert C escape characters into two characters '\\' and the other character
```

Comment:
I'd appreciate a colon after 'two characters' above

Reply:
Colon added.

With the name `addUnescapedChar` I was trying to convey that it takes a possibly escaped character and turns it into something without escape characters.  "Unescaped" was meant to imply the breaking up of escape characters.  I can use your suggested name though.